src/Api.php

```diff
@@ -128,6 +128,8 @@ public function __construct(
 
         if (!isset($http_client)) {
             $http_client = new GClient();
+            $http_client->setDefaultOption('timeout', 30);
```

Comment:
Some API calls may take up to 5min. While this is an edge case, it may impact some of the developers needing the most an API. As the connect_timeout is a separate parameter, I'd increase it.